source/localizable/_code-of-conduct.en.md

```diff
@@ -1,39 +1,39 @@
 Purpose
 -------
 
-A primary goal of all the conferences and user groups that refer to this Code of Conduct is to be inclusive to the largest number of contributors, with the most varied and diverse backgrounds possible. As such, we are committed to providing a friendly, safe and welcoming environment for all, regardless of gender, sexual orientation, ability, ethnicity, socioeconomic status and religion (or lack thereof).
+A primary goal of all the conferences and user groups that follow this Code of Conduct is to create an inclusive community that welcomes and supports as many people with various backgrounds and identities as possible. As such, we are committed to build a friendly and safer environment for all. These various backgrounds and identities include (but are not limited to) gender identity and expression, sex, sexual orientation, educational level, mental and physical ability, immigration status, race, ethnicity, national origin, social and economic status, age, size and religion (or lack thereof).
```

Comment:
I like the old wording with `regardless of ...` better, the overall sentence is rather long but to me it flows better and has less repetition.

I'm also not sure about providing vs. building, overall what we try to do is build a better community, that's right - but for an individual event we try to provide one. To me, build fits better for the purpose of an association and build fits better for a coc related to events.

I might be missing something here but I don't know why we'd need race, ethnicity and national origin. I thought `ethnicity` rather covers that.

Also not sure which meaning of size this would translate to :)